.dockerignore

```diff
@@ -1 +1,6 @@
-frontend/node_modules
+**/node_modules
+.git
+app/dist
+app/build
```

Comment:
Maybe we should ignore the whole app as it's not used for docker.

Reply:
This done.